### ISSUE TITLE
Move files using the title bar

### ISSUE THREAD

**Issue by BenoitDuffez** [elementary/files#986]
My request is about Evince. I would have submitted it there, but I feel this is more related to its integration into Elementary OS. I'm not sure though, so please bear with me and indicate me if this is not the right place.

I love Elementary on my work laptop, and I use a mac at home. What I love from both is the ability to quickly open PDF files with a fast viewer. What I love on the mac and I feel is missing on Elementary, is the ability to rename/move the file from the title bar of the PDF viewer.

Here's how it looks like on a mac (screenshot taken randomly from Google Images):

![rename and move files from the title bar](http://cdn.osxdaily.com/wp-content/uploads/2014/03/move-a-file-while-open-from-titlebar.jpg)

Basically the title bar contains the file name, (right) clicking it opens a popup allowing to rename the file, and a quick file browser to move the file to recent folders, and a button to open a full folder browser to place it in a non-recent folder.

This is very useful and helps productivity, e.g. when someone sends you a document opened from another app (likely in /tmp/mozilla/whatever) and you want to save it to $HOME/Documents/Products/etc

**Comment by peteruithoven** [comment]
I like this idea, but I don't think this is something we can do system wide, but individual applications are free to do so. One example I'd like to mention is Notejot, where title in the header is also the place to rename the note. 

My recommendation would be to open feature requests for specific applications where this could be useful like Code or Photos, these are apps we have a lot of control over. You can of course also open one for Evince, but it's a gnome project, repo: https://gitlab.gnome.org/GNOME/evince).

For now I'm closing this one as out of scope, please do open those more specific feature requests. I've also moved the issue to the OS which is kind of our catch all place for issues.
